be/src/Project/Services/Ordering/Ordering.Api/v1/RequestModels/OrderItemStatusCommentRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Ordering.Api.v1.RequestModels
+{
+    public class OrderItemStatusCommentRequestModel
+    {
+        public Guid? OrderItemId { get; set; }
+        public string OrderItemStatusComment { get; set; }
```

Comment:
**suggestion:** there are language params missing everywhere. You'll never know what language the comment has been provided in.